pages/privacy.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: page-fullwidth
+title: "HPC Carpentry Privacy Policy"
+permalink: /privacy/
+---
+
```

Comment:
This needs work. The privacy policy relates to information we collect and store, which we have full control over. Currently, we do not collect any sensitive personally identifiable information; we ask for names of instructors & helpers, but no email or social media handles, so those can be considered ambiguous. Let's spell out this stance, and for now, *not* adhere to The Carpentries' policy.

Reply:
Ok, I made a stab at it.